controllers/users/users.js

```diff
@@ -28,7 +28,7 @@ exports.getUser = async (req, res) => {
       .select("-activationKey")
       .select("-updatedAt")
       .select("-__v");
-    res.json(user);
+    res.json({ user: user});
```

Comment:
if the key and value has the same name, you can replace it to 
`{ user }` instead of `{ user: user }`